src/vivarium_nih_us_cvd/components/effects.py

```diff
@@ -107,25 +98,13 @@ def _polypill_sbp_adherence_modifier(
 class PAFCalculationRiskEffect(RiskEffect):
     """Risk effect component for calculating PAFs"""
 
-    def __repr__(self):
-        return f"PAFCalculationRiskEffect(risk={self.risk}, target={self.target})"
-
-    ##############
-    # Properties #
-    ##############
-
-    @property
-    def name(self) -> str:
-        return f"paf_calculation_risk_effect.{self.risk}.{self.target}"
+    def get_population_attributable_fraction_source(
+        self, builder: Builder
+    ) -> Optional[LookupTable]:
+        return None
 
-    #################
-    # Setup methods #
-    #################
-
-    # noinspection PyAttributeOutsideInit
-    def setup(self, builder: Builder) -> None:
```

Comment:
These methods look undefined - how was this even working?

Reply:
It inherits from `RiskEffect`, so these methods are defined. Why do you think they aren't?